ros2_controllers_test_nodes/setup.py

```diff
@@ -25,8 +25,7 @@
     data_files=[
         ("share/ament_index/resource_index/packages", ["resource/" + package_name]),
         ("share/" + package_name, ["package.xml"]),
-        ("share/" + package_name, glob("launch/*.launch.py")),
-        ("share/" + package_name + "/configs", glob("configs/*.*")),
+        ("share/" + package_name + "/test", glob("test/*.yaml")),
```

Comment:
@christophfroehlich is the *.py inside the test folders not needed here?

Reply:
the launch/ and configs/ folder never existed ;) and pytest uses automatically all python scripts in the test folder, i.e., the tests run with the current setup:
https://github.com/ros-controls/ros2_controllers/actions/runs/11888384891/job/33122823811?pr=1369#step:5:12345

Reply:
Perfect